### DISCUSSION TITLE
Documentation says nothing about exernal network "net" needed for Docker.... help?

### DISCUSSION THREAD

**Discussion by F1zzyD** [authelia/authelia#8512] [Q&A] [answered]
As a new user following the documentation (which the config file says to read over and over again), the documentation says the following:

> The following examples are [Docker Compose](https://docs.docker.com/compose/) deployments with just Authelia and no bundled applications or proxies.
> 
> It expects the following:
> 
> The file data/authelia/config/configuration.yml is present and the configuration file.
>     The directory data/authelia/secrets/ exists and contain the relevant [secret](https://www.authelia.com/configuration/methods/secrets/) files:
>  A file named JWT_SECRET for the [jwt_secret](https://www.authelia.com/configuration/identity-validation/reset-password/#jwt_secret)
>  A file named SESSION_SECRET for the [session secret](https://www.authelia.com/configuration/session/introduction/#secret)
>  A file named STORAGE_PASSWORD for the [PostgreSQL password secret](https://www.authelia.com/configuration/storage/postgres/#password)
> A file named STORAGE_ENCRYPTION_KEY for the [storage encryption_key secret](https://www.authelia.com/configuration/storage/introduction/#encryption_key)
> You’re using PostgreSQL.
> **You have an external network named net which is in bridge mode.**

The last requirement in the documentation of an external network named `net` is not detailed it's purpose, how to set it up, and has no other reference in the documentation. As a user trying to set up Authelia for the first time, I am closed off from setting up this project without figuring out what `net` is needed for. Docker will not allow me to at least test my configuration without this unexplained detail.

For example, I am trying to use Authelia as a login for ConvertX which is also running in Docker. I have made the JWT_SECRET, SESSION_SECRET, STORAGE_PASSWORD, and the STORAGE_ENCRYPTION_KEY along with including the pre-made docker-compose layout in my ConvertX compose.yml. Again, I am unable to test to see if this works without the required `net`. 

Any help on this? It's missing from the documentation.

**Comment by hendrik1120** [comment, marked as answer]
Hi, thanks for trying out authelia.
The documentation assumes prior knowledge of using docker. There is currently an effort being made to make the documentation more accessible for new users, but it won't be possible to include a full docker guide.
Thanks for pointing out that section in the docs, it's indeed not easy to understand.

In order to create the mentioned network, use the `docker network create net` command.
Please also use docker bind mounts to make your configuration permanent, which should solve your issue #8513

If you need any help setting up authelia, please stop creating more issues and use this discussion or reach out on discord.
